app/assets/javascripts/select-kit/components/icon-picker.js.es6

```diff
@@ -0,0 +1,49 @@
+import MultiSelectComponent from "select-kit/components/multi-select";
+import { computed } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { makeArray } from "discourse-common/lib/helpers";
+
+export default MultiSelectComponent.extend({
+  pluginApiIdentifiers: ["icon-picker"],
+  classNames: ["icon-picker"],
+
+  content: computed("value.[]", function() {
+    return makeArray(this.value).map(this._transformIconName);
+  }),
+
+  search(filter) {
+    return ajax("/svg-sprite/all-icons", { data: { filter } }).then(icons =>
+      icons
+        .filter(icon => !makeArray(this.value).includes(icon))
+        .map(this._transformIconName)
+    );
+  },
+
+  actions: {
+    onChange(value, item) {
+      if (this.selectKit.options.maximum === 1) {
+        value = value.length ? value[0] : null;
+        item = item.length ? item[0] : null;
+      }
+
+      this.attrs.onChange && this.attrs.onChange(value, item);
+    }
+  },
+
+  _transformIconName(icon) {
+    let processedIconName = icon;
+    if (!icon.startsWith("fa")) {
+      processedIconName = `fa-${processedIconName}`;
```

Comment:
I'm a bit torn about including the "fa-" prefix in the displayed icon name here. FontAwesome, when you search for icons, doesn't display the prefix. I think we should only display the `fab-` and `far-` prefixes in the picker.

Reply:
so value should have fa- but we never display it, right?

Reply:
No, we don't need `fa-` at all, only `fab-` (for brand icons) and `far-` (for the regular icon set).